run.go

```diff
@@ -67,6 +67,12 @@ func execContainer(context *cli.Context, spec *specs.LinuxSpec) (int, error) {
 // default action is to execute a container
 func runAction(context *cli.Context) {
 	spec, err := loadSpec(context.Args().First())
+
+	if os.Getenv("NOTIFY_SOCKET") != "" {
```

Comment:
Could you store the value in a variable so we make the os.Getenv call only once instead of 4 times?